include/ozo/transaction_options.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <boost/hana/optional.hpp>
+#include <boost/hana/type.hpp>
+#include <ozo/core/none.h>
+#include <ozo/core/options.h>
+#include <type_traits>
+
+namespace ozo {
+
+namespace hana = boost::hana;
+
+/**
+ * @ingroup group-connection-types
+ * @brief 'type enum' for transaction isolation levels supported by PostgreSQL
```

Comment:
I'd like to ask you to provide for this and the other options a link for the related PostgreSQL documentation here, to allow a user easily navigate to it.